pkg/client/clientset_generated/internalclientset/typed/batch/unversioned/batch_client.go

```diff
@@ -81,9 +82,17 @@ func setConfigDefaults(config *restclient.Config) error {
 	}
 	// TODO: Unconditionally set the config.Version, until we fix the config.
 	//if config.Version == "" {
+	// XXX why is above commented out?
 	copyGroupVersion := g.GroupVersion
 	config.GroupVersion = &copyGroupVersion
 	//}
+	// Do we need something like this:
```

Comment:
I remember that line now. We never properly fix the setConfigDefaults because we haven't had multiple versions for a long time. I'll take a look later today. If it's urgent, you can try @krousey to see if he can get to it earlier than me.